client/app/license/redirection/upgrade/license-redirection-upgrade.controller.js

```diff
@@ -1,12 +1,21 @@
-angular.module("Module.license").controller("ConfirmUpgradeRedirectionCtrl", ($location, $scope, $timeout) => {
-    "use strict";
+angular.module("Module.license").controller("ConfirmUpgradeRedirectionCtrl", class ConfirmUpgradeRedirectionCtrl {
```

Comment:
Could you also rename controller like: `ConfirmUpgradeRedirectionCtrl` => `LicenseRedirectionUpgradeCtrl`?